src/main/java/com/chargerrobotics/subsystems/LimelightSubsystem.java

```diff
@@ -68,6 +71,15 @@ public double getV() {
 		return v;
 	}
 
+	// distance in inches
+	public double distance() {
+		if (v == 0) {
+			return -1.0;
+		} else {
+			return ((Constants.targetHeight - Constants.cameraHeight)/Math.tan(((Constants.cameraAngle + y)*Math.PI)/180));
```

Comment:
`Math.toRadians(double angdeg);` would make it much easier to read and [slightly more accurate](https://bugs.openjdk.java.net/browse/JDK-8051808) than manually converting.